backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -688,3 +688,104 @@ def test_cancel_workshop_attendance(self):
+        self.request_review_time = self.project_1.start_time + timedelta(1)
+        self.project_1.request_review(force_timestamp=self.request_review_time)
+        self.time_one = self.project_1.start_time - timedelta(days=6)
+        self.time_two = self.project_1.start_time + timedelta(days=4)
+        self.time_three = self.project_1.start_time + timedelta(days=3)
+        self.time_four = self.project_1.start_time + timedelta(days=2)
+
+    def test_get_users_that_reviewed_since_last_review_request_for_project(self):
+
+        # Four reviews are made with the four review times above (No reviews done on project_2)
+        review_1 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+            timestamp=self.time_one
+        )
+        review_1.timestamp = self.time_one
+        review_1.save()
+
+        review_2 = factories.RecruitProjectReviewFactory(
+            status=COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_2.timestamp = self.time_two
+        review_2.save()
+
+        review_3 = factories.RecruitProjectReviewFactory(
+            status=EXCELLENT,
+            recruit_project=self.project_1,
+        )
+        review_3.timestamp = self.time_three
+        review_3.save()
+
+        review_4 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_4.timestamp = self.time_four
+        review_4.save()
+
+        result = self.card_1.get_users_that_reviewed_since_last_review_request()
+        ids_which_can_be_returned = [review_2.reviewer_user.id, review_3.reviewer_user.id, review_4.reviewer_user.id]
+
+        # Making sure that reviews were done on card and not on card_2, if card_2 returns reviews then our function is
+        # returning the wrong stuff and therefore it is not working as it should.
+        self.assertEqual(sorted(result), sorted(ids_which_can_be_returned))
+        assert len(self.card_1.get_users_that_reviewed_since_last_review_request()) > 0
+
+    def test_get_users_that_reviewed_since_last_review_request_for_project_2(self):
+
+        # Create a review request on card_2 of project_2
+        request_review_time_2 = self.project_2.start_time + timedelta(1)
+        self.project_2.request_review(force_timestamp=request_review_time_2)
+        time_1 = self.project_2.start_time - timedelta(days=10)
+
+        # Create a review on card_2, the review will be before the latest request for a review
+        review_1_card_2 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_2,
+            timestamp=time_1
+        )
+        review_1_card_2.timestamp = time_1
+        review_1_card_2.save()
+
+        # Review done on project_2, card_2 were done before the latest request for a review, therefore,
```

Comment:
You already tested the timing in the other test. What is this test for? Is it just a repeat?